cli/internal/daemonclient/daemonclient.go

```diff
@@ -0,0 +1,53 @@
+// Package daemonclient is a wrapper around a grpc client
+// to talk to turbod
+package daemonclient
+
+import (
+	"context"
+
+	"github.com/vercel/turborepo/cli/internal/server"
+)
+
+// DaemonClient provides access to higher-level functionality from the daemon to a turbo run.
+type DaemonClient struct {
+	client server.TurboClient
```

Comment:
We can, and maybe should, move the autogenerated code to a different, more neutrally-named package. Then this would be `client turboprotocol.TurboClient` or something like that. Open to suggestions on the package name.

Reply:
`turbodprotocol` it is. Along with ignoring the generated files and adding a task to compile them on folder-open.